types/react-syntax-highlighter/index.d.ts

```diff
@@ -20,6 +20,13 @@ declare module 'react-syntax-highlighter' {
         showLineNumbers?: boolean;
         startingLineNumber?: number;
         lineNumberStyle?: any;
+        showInlineLineNumbers?: boolean;
+        lineNumberContainerStyle?: any;
+        wrapLines?: boolean;
+        wrapLongLines?: boolean;
+        renderer?: any;
+        PreTag?: any;
+        CodeTag?: any;
```

Comment:
For `PreTag` and `CodeTag`, refer to this section from the readme: https://github.com/react-syntax-highlighter/react-syntax-highlighter#use
> PreTag - the element or custom react component to use in place of the default pre tag, the outermost tag of the component (useful for custom renderer not targeting DOM).

So the valid types are any element or React component. How would we type that?

I used `styled-components` as a reference, and they do so like this: 
```JSX.IntrinsicElements | React.ComponentType<any>```
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/styled-components/index.d.ts#L193